src/instructions.spec.ts

```diff
@@ -1141,6 +1141,34 @@ describe('Cortex-M0+ Instruction Set', () => {
     expect(registers.V).toEqual(false);
   });
 
+  it('should execute a `rsbs r0, r3` instruction', async () => {
+    // This instruction is also calledasync  `negs`
+    await cpu.setPC(0x20000000);
+    await cpu.writeUint16(0x20000000, opcodeRSBS(r0, r3));
+    await cpu.setRegisters({ r3: 0x80000000 });
+    await cpu.singleStep();
+    const registers = await cpu.readRegisters();
+    expect(registers.r0).toEqual(0x80000000);
+    expect(registers.N).toEqual(true);
+    expect(registers.Z).toEqual(false);
+    expect(registers.C).toEqual(false);
+    expect(registers.V).toEqual(true);
+  });
+
+  it('should execute a `rsbs r0, r3` instruction', async () => {
+    // This instruction is also calledasync  `negs`
```

Comment:
same here :-)

Reply:
OK no problem, finally I realized that all sub related instructions , even immediate can use the same code, if You agree next commit will include them.
I don't think it is worth adding more test as I expect a duplicated behavior

Reply:
Yes, that sounds fine!